### PR TITLE
FIX #673 - CardFooter component

### LINKED ISSUE #673: [FEATURE] Create new CardFooter component
**Describe the solution you'd like**
A simple wrapper component for the Card footer.

### PR DESCRIPTION
**Pull request by luco5826**
Fixes #673 

#### PR Checklist

- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
The new CardFooter component is a wrapper for the class `it-card-footer` provided by BI.

#### Changes proposed in this pull request:

- New component in Cards' storybook (showing the CardFooter usage)

### CODE REVIEW

**Comment by dej611** [comment]
Hi @luco5826 , thanks for your first contribution!

I'm reviewing this PR but I'm a bit concerned with the renaming bit. I guess the original issue wasn't clear enough, so I'm sorry for that.
Renaming a component may be a breaking change - as small as it could be - in this context, so I would probably ask if it's possible to restore the previous version of the component `CardFooterCTA` ( bad name, I agree on that ) and keep the story, maybe with a better name? Something like "`Card con footer`": what do you think?

**Comment by luco5826** [comment]
Hi @dej611 !

You're completely right, I didn't think about the renaming being a breaking change and in fact I was a bit worried about that. 

I'm going to fix it asap and since we're here, do you think it is better to create a CardFooter component as I implemented it (as a simple `div`) or as a wrap around  the existing `CardFooter` component from `reactstrap`?

File: src/components/Card/CardFooterCTA.js [hunk 1]
```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types'
...
+  className: PropTypes.string,
+  children: PropTypes.node
 }
 
 const CardFooterCTA = props => {
-  const { className, ...attributes } = props
+  const { className, children } = props
   const classes = classNames(className, 'it-card-footer')
-  return <div className={classes} {...attributes} />
+  return <div className={classes}>{children}</div>
```
Comment (dej611) [inline src/components/Card/CardFooterCTA.js:13]: Can you restore it with the old version?

The problem with the new one is that you are only letting the `children` props pass. Say the developer wants to add an `aria-something` on the `div`, how can he does that?

`attributes` contains `children` and also many HTML attributes in it.

**Review by dej611 — CHANGES_REQUESTED** [review]
Left a note.
Everything else is ok, but I need that change to approve it 👍

**Comment by luco5826** [comment]
@dej611 Hi! 
I've restored the previous version of `CardFooterCTA` but atm I really do not understand what was the point about this issue. 
The component already existed and it was implemented in the correct way, so the only feature was to add it to the documentation, right?

**Comment by dej611** [comment]
Yes. True.
As said above the original issue was misleading and not well explained.
I think your work on the documentation is great and will definetly help developers!

**Review by dej611 — APPROVED** [review]

**Pull request merged**